client/src/components/BurgerMenu.jsx

```diff
@@ -11,7 +11,7 @@ Modal.setAppElement("#root");
 
 function Burger() {
   const [isOpen, setIsOpen] = useState(false);
-  const auth = useAuth();
+  const { token, logOut } = useAuth();
```

Comment:
Pulled the logOut function and token here so you don't have to run `useAuth()` twice